Docs.Site/Docs/Reference/Symbols.markdown

```diff
@@ -51,6 +90,20 @@ The package details page for a package that has symbols available comes with a n
 
 Here's a quick cheatsheet of the commands related to symbol feeds:
 
+### New .snupkg symbols
+* Create package and new symbols
+    
+    ```nuget.exe pack <path_to_project_or_nuspec> -Symbols -SymbolPackageFormat snupkg```
+    
+* Pushing a package to MyGet (we will detect the .snupkg and push it):
+
+    ```nuget.exe push <package-file> <myget-key> -Source https://www.myget.org/F/<feed-name>/api/v3/index.json```
```

Comment:
I think <myget-key> should be <access-token> or <apiKey>, we need to check what we use in docs

Reply:
@matthcap Sorry, for this strange comment, I wanted to say that we should change <myget-key> to <access-token> or <API-key>, because iirc we never were using <myget-key>